Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -865,140 +877,147 @@ extension PaymentSheetUITest {
-        _testUSBankAccount(mode: "Setup", integrationType: "Def SSC")
-    }
-
-/* Disable Link test
-    func testDeferredIntentLinkSignIn_ClientSideConfirmation() throws {
-        loadPlayground(
-            app,
-            settings: [
-                "customer_mode": "new",
-                "automatic_payment_methods": "off",
-                "link": "on",
-                "init_mode": "Deferred",
-            ]
-        )
-
-        app.buttons["Checkout (Complete)"].tap()
-
-        let payWithLinkButton = app.buttons["Pay with Link"]
-        XCTAssertTrue(payWithLinkButton.waitForExistence(timeout: 10))
-        payWithLinkButton.tap()
-
-        try loginAndPay()
-    }
-*/
-/* Disable Link test
-    func testDeferredIntentLinkSignIn_ClientSideConfirmation_LostCardDecline() throws {
-        loadPlayground(
-            app,
-            settings: [
-                "customer_mode": "new",
-                "automatic_payment_methods": "off",
-                "link": "on",
-                "init_mode": "Deferred",
-            ]
-        )
-
-        app.buttons["Checkout (Complete)"].tap()
-
-        let payWithLinkButton = app.buttons["Pay with Link"]
-        XCTAssertTrue(payWithLinkButton.waitForExistence(timeout: 10))
-        payWithLinkButton.tap()
-
-        try linkLogin()
-
-        let modal = app.otherElements["Stripe.Link.PayWithLinkViewController"]
-        let paymentMethodPicker = app.otherElements["Stripe.Link.PaymentMethodPicker"]
-        if paymentMethodPicker.waitForExistence(timeout: 10) {
-            paymentMethodPicker.tap()
-            paymentMethodPicker.buttons["Add a payment method"].tap()
-        }
-
-        try fillCardData(app, container: modal, cardNumber: "4000000000009987")
-
-        let payButton = modal.buttons["Pay $50.99"]
-        expectation(for: NSPredicate(format: "enabled == true"), evaluatedWith: payButton, handler: nil)
-        waitForExpectations(timeout: 10, handler: nil)
-        payButton.tap()
-
-        let failedText = modal.staticTexts["The payment failed."]
-        XCTAssertTrue(failedText.waitForExistence(timeout: 10))
-    }
-*/
-/* Disable Link test
-    func testDeferredIntentLinkCustomFlow_ClientSideConfirmation() throws {
-        loadPlayground(
-            app,
-            settings: [
-                "customer_mode": "new",
-                "automatic_payment_methods": "off",
-                "link": "on",
-                "init_mode": "Deferred",
-            ]
-        )
-
-        let paymentMethodButton = app.buttons["Select Payment Method"]
-        XCTAssertTrue(paymentMethodButton.waitForExistence(timeout: 10.0))
-        paymentMethodButton.tap()
-
-        let addCardButton = app.buttons["Link"]
-        XCTAssertTrue(addCardButton.waitForExistence(timeout: 10.0))
-        addCardButton.tap()
-
-        app.buttons["Checkout (Custom)"].tap()
+        _testUSBankAccount(mode: .setup, integrationType: .deferred_ssc)
+    }
+
+    /* Disable Link test
```

Comment:
Same comment as above -- either delete completely, or update the settings to use the new format?